packages/quick_actions/quick_actions/example/android/app/build.gradle

```diff
@@ -53,6 +53,7 @@ flutter {
 
 dependencies {
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
+    api 'androidx.test:core:1.2.0'
```

Comment:
do you know why this requires api instead of `androidTestImplementation`? https://developer.android.com/training/testing/set-up-project#add-gradle-dependencies

Reply:
same for the other ones

Reply:
This was the installation requirements for the [espresso](https://pub.dev/packages/espresso#installation) package. I wasn't able to get it to work with `androidTestImplementation`. I'm also not familiar enough with gradle to understand why its `api` without doing some more research.

Reply:
I see. this is probably due to https://github.com/flutter/flutter/issues/56591